modules/ROOT/pages/indexes/semantic-indexes/vector-indexes.adoc

```diff
@@ -648,36 +451,31 @@ The types are still enforced as `LIST<INTEGER \| FLOAT>`.
 | There is no Cypher syntax for creating a vector index.
 
 [TIP]
-====
 Use the procedure `db.index.vector.createNodeIndex` to create the a vector index.
 Procedure signature:
 [source,syntax,role="noheader"]
 ----
 db.index.vector.createNodeIndex(indexName :: STRING, label :: STRING, propertyKey :: STRING, vectorDimension :: INTEGER, vectorSimilarityFunction :: STRING)
 ----
-====
```

Comment:
without this grouping it looks very odd
![image](https://github.com/neo4j/docs-cypher/assets/7591575/2df7e14f-0b32-4f3c-8ccc-4f909261baa5)
vs the old
![image](https://github.com/neo4j/docs-cypher/assets/7591575/e227d0fb-6e04-4b78-a1b1-6ecaceb9f7a4)
which clearly showed the signature belonging to the tip and not freestanding after (especially since the end of the ip says `Procedure signature:`)

Reply:
indeed, this was the original intent of the code block with the tip, and why we wrote it as such.